CONTRIBUTING.md

```diff
@@ -1,31 +1,149 @@
 # How To Contribute
 
-## Installation
+- [Overview](#overview)
+- [Clone the repo](#clone-the-repo)
+- [Install dependencies](#install-dependencies)
+- [Adding components](#adding-components)
+- [Running the docs/test-app](#running-the-docstest-app)
+- [Viewing changes in the docs/test-app](#viewing-changes-in-the-docstest-app)
+- [Tests](#tests)
+- [Documentation](#documentation)
+- [VSCode setup](#vscode-setup)
+  - [Glint](#glint)
+  - [VSCode recommended extensions](#vscode-recommended-extensions)
 
-* `git clone <repository-url>`
-* `cd ember-toucan-core`
-* `pnpm install`
+## Overview
 
-## Linting
+The ember-toucan-core repo is an [Ember v2 addon](https://rfcs.emberjs.com/id/0507-embroider-v2-package-format/). It is setup as a monorepo with the following structure:
 
-* `pnpm lint`
-* `pnpm lint:fix`
+- `ember-toucan-core` - the actual Ember addon containing the Toucan components
+- `docs-app` - an Ember application used for the documentation site at http://ember-toucan-core.pages.dev
+- `docs` - contains markdown files that are consumed by the `docs-app`
+- `test-app` - the test app where tests are written
 
-## Building the addon
+## Clone the repo
 
-* `cd ember-toucan-core`
-* `pnpm build`
+The first step in developing in the ember-toucan-core repo is to clone the repo.
 
-## Running tests
+```bash
+git clone git@github.com:CrowdStrike/ember-toucan-core.git
+```
 
-* `cd test-app`
-* `pnpm test` – Runs the test suite on the current Ember version
-* `pnpm test:watch` – Runs the test suite in "watch mode"
+## Install dependencies
 
-## Running the test application
+ember-toucan-core uses [pnpm](https://pnpm.io) as a package manager. If you're used to yarn or npm, pnpm has the identical commands so you should be right at home!
 
-* `cd test-app`
-* `pnpm start`
-* Visit the test application at [http://localhost:4200](http://localhost:4200).
+```bash
+cd ember-toucan-core
+pnpm install
+```
 
-For more information on using ember-cli, visit [https://cli.emberjs.com/release/](https://cli.emberjs.com/release/).
+## Adding components
+
+When developing a new component, add it to the `ember-toucan-core` directory. A few things to remember:
+
+1. Components are added under `ember-toucan-core/src/components`
+2. When adding a new component it needs to be added to the template registry so that Glint picks it up. To do that, add a line in `ember-toucan-core/src/template-registry.ts`.
```

Comment:
nit: 

```suggestion
2. When adding a new component, it needs to be added to the template registry so that Glint picks it up. To do that, add a line in `ember-toucan-core/src/template-registry.ts`.
```